using-valtimo/access-control/access-control.md

```diff
@@ -1,7 +1,30 @@
 # Access control
 
-Access control allows application functionality to be enabled or restricted through policies. 
+Valtimo supports Policy Based Access Control (PBAC) as an authorization layer via this module.
+This layer allows you to restrict access to functionality or data based on policies.
 
-This section contains information on how to configure and use access control.
+## Roles
+A role is a simple entity where permissions can be linked to. The role names should match the roles in the JWT token by OpenId/OAuth2 (Keycloak).
 
+## Permissions
+Permissions are quite a bit more complex than the roles.
+
+Multiple permissions are evaluated as OR:
+The outcome of a permission (action) check is true when ANY of the permissions passes its checks.
+When permissions are used to filter data from a list query, the result will contain every entry that passes the checks of ANY permission.
+
+When selecting applicable permissions for evaluation, the following checks are performed:
+- The user has the role that is configured for the permission
+- The requested resource matches the permissions' resource
+- The required action matches the permissions' action 
+
+## Conditions
+Extra conditions can be applied to a permission to tighten the access to resources. These conditions are optional.
+Conditions are evaluated as AND, which means the ALL configured conditions on a permission should evaluate to true to pass the permission checks.
+
+Conditions can also be nested by using a container. This allows you to join another resource to the root resource, on which you can add conditions as well.
```

Comment:
Can you add an example and reason as to why you would want to do this?

Reply:
I added a reference to what was already there.

Reply:
Nice! I would still like to see a reason as to why you would want to use conditions on this page, though.